server_admin/topics/sessions/offline.adoc

```diff
@@ -21,7 +21,8 @@ The admin can also view all the offline tokens issued in the `Offline Access` ta
 Offline tokens can also be revoked by setting a <<_revocation-policy, revocation policy>>.
 
 To be able to issue an offline token, users need to have the role mapping for the realm-level role `offline_access`.
-Clients also need to have that role in their scope.
+Clients also need to have that role in their scope. Finally client needs to have an `offline_access` client scope added as an `Optional
+client scope` to it, which is by default.
```

Comment:
Clients also need to have that role in their scope. Finally, the client needs to have an `offline_access` client scope added as an `Optional client scope` to it, which is done by default.